lewis_emulators/knr1050/interfaces/stream_interface.py

```diff
@@ -81,6 +88,7 @@ def stop_klv(self):
         return 'OK'
 
     @if_connected
```

Comment:
I would expect this to look something like:

```suggestion
    @conditional_reply("connected")
```

Reply:
As far as I'm aware this is the standard use of `@if_connected`. In the emulator path I can see 10+ devices that use this same decorator.

The use of conditional reply her is that we can define the type of reply in the preamble and then decorate the functions you want to test against.